apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-card.js

```diff
@@ -69,7 +69,7 @@ function WelcomeTourCard( {
 							isTertiary
 							onClick={ () => setCurrentCardIndex( 0 ) }
 						>
-							Restart tour
+							{ __( 'Restart tour', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **12** times:
`translate( 'Restart Tour' )` *ES Score: 14*